integration-test/assets/artifact.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I think it would be better to leave these in the action repo. The whole point of that action is to avoid having a copy in each repository that runs cli integration tests.